[suspendmanager] don't unsuspend constructions blocked by an item (#4454)
DFHack/dfhack: docs/changelog.txt

- fix behavior of Linux Steam launcher on systems that don't support the inotify API
- fix rendering of resize "notch" in lower right corner of resizable windows in ascii mode
- `quickfort`: stockpiles can now be placed even if there is water covering the tile, as per vanilla behavior

> - `suspendmanager`: fix a cancellation spam when an item prevents a building to be completed

Reviewer: ```suggestion
- `suspendmanager`: prevent cancellation spam when an item is preventing a building from being completed
```
